packages/yarnpkg-core/sources/Configuration.ts

```diff
@@ -374,6 +374,14 @@ export const coreDefinitions: {[coreSettingName: string]: SettingsDefinition} =
       type: SettingsType.ANY,
     },
   },
+
+  // Node modules-specific setttings
+  nmHoistBorders: {
```

Comment:
Is there any benefit to shortening `nodeModules` inside configuration options? Searching for "nm" in the documentation only returns 2 entries on the telemetry page. 🤔

Reply:
We have `pnpMode`, so having `nmSomething` looks logical from this point of view